src/constants.ts

```diff
@@ -164,6 +164,6 @@ var Constants = {
         EventBatchingIntervalMillis: 'eventBatchingIntervalMillis',
     },
     DefaultInstance: 'default_instance',
-};
+} as const;
```

Comment:
Is this `as const` necessary?

Reply:
Actually yeah. This is a `Const Assertion` It helps the compiler know for certain this is a constant with a set number of options and should be read only. It's perfect for what we're trying to do here.